Node JS/session fswd 01/UI/food-adda/src/pages/PreviousOrders.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react';
+
+{/* hemant -previousorder*/}
+
+const PreviousOrders = () => {
+    useEffect(() => {
+        const orderItems = document.querySelectorAll('.order-item');

```

Comment:
implement onClick in divs, instead of using js selectors

Reply:
Done